README.md

```diff
@@ -86,12 +89,14 @@ intermediate management tool responsible for high availability and failover
 management like similar open source operators.
 
 CloudNativePG also manages additional resources to help the `Cluster` resource
-manage PostgreSQL - currently `ScheduledBackup`, `Pooler`, and `Backup`.
+manage PostgreSQL - currently `Backup`, `ClusterImageCatalog`, `ImageCatalog`,
+`Pooler`, and `ScheduledBackup`.
 
-Fully embracing Kubernetes also means that, in case of failure of the whole
-Kubernetes cluster, the operator won’t do anything, postponing any decision to
-when the cluster is back up again. In the meantime, Postgres instances should
-continue running according to the last known state of the cluster.
+Fully embracing Kubernetes involves a hands-off approach during temporary
```

Comment:
Not loving "involves" here. I think the previous version with "means" is better.
```suggestion
Fully embracing Kubernetes means adopting a hands-off approach during temporary
```